benchmarks/online-shop/recommendationservice/requirements.txt

```diff
@@ -17,24 +17,24 @@ google-api-core[grpc]==1.31.5
     #   google-cloud-core
     #   google-cloud-monitoring
     #   google-cloud-trace
-    #   google-python-cloud-debugger
+    #   #google-python-cloud-debugger
```

Comment:
remove the commented stuff?

Reply:
I removed now all the cloud-debugger dependencies.